AEPMessaging.xcodeproj/project.pbxproj

```diff
@@ -783,7 +842,7 @@
 			buildActionMask = 2147483647;
 			files = (
 				091881FB2A16D15100615481 /* AEPMessaging.framework in Frameworks */,
-				8DEEA93D16C40CA285E20AD5 /* Pods_MessagingDemoAppSwiftUI.framework in Frameworks */,
+				11809F1839C1AFE69C2A61CA /* Pods_MessagingDemoAppSwiftUI.framework in Frameworks */,
```

Comment:
Plenty of ID changed as a result of running
pod deintegrate and pod install.

Not much changed otherwise